frontend/src/components/user-profile/user-profile.tsx

```diff
@@ -33,10 +38,37 @@ const UserPage: React.FC = () => {
     isFollowed: userProfile.isFollowed,
     followersCount: userProfile.followersCount,
     followersDataStatus: userProfile.followersDataStatus,
+    favoriteEpisodes: userProfile.favoriteEpisodes,
+    favoriteEpisodesCount: userProfile.favoriteEpisodesTotalCount,
   }));
 
   const dispatch = useDispatch();
 
+  const [episodePagination, setEpisodePagination] = useState(DEFAULT_EPISODE_PAGINATION);
+
+  useEffect(() => {
+    if(currentUser?.id === user?.id) {
```

Comment:
Maybe let's put it to `isOwner` variable?

Reply:
agreed